website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -212,10 +200,66 @@ There are a number of other useful configurations that can be made when initialz
 - Filter watch events using [predicates][event_filtering]
 - Choose the type of [EventHandler][event_handler_godocs] to change how a watch event will translate to reconcile requests for the reconcile loop. For operator relationships that are more complex than primary and secondary resources, the [`EnqueueRequestsFromMapFunc`][enqueue_requests_from_map_func] handler can be used to transform a watch event into an arbitrary set of reconcile requests.
 
+### Reconcile loop
+
+The reconcile function is responsible for synchronizing the desired state as represented by their Specs and the actual state of the system. In this way, it works like a loop, and it does not stop until all conditionals match its implementation. The following is pseudo-code with an example that clarifies it.
+
+```go
+reconcile App {
```

Comment:
This should be valid Go code.

Reply:
The goal here is to demonstrate how it works. That it is a loop. a valid go code will not pass the idea easily. So, I think is better to remove that.